README.md

```diff
@@ -4,10 +4,10 @@
  
 # pretty-print - pp
 
-Adds a `pp(Any?)` and `T.pp()` method to pretty print any Java or Kotlin object.
+Adds a `pp(Any?)` and `T.pp()` method to pretty print Java or Kotlin object.
 
-`pp(Any?)` takes any object and will print it in a pretty format.
-`T.pp()` can be called on any object inline and will allow you to pretty print inside of a method chain.
+`pp(Any?)` takes an object and prints in a pretty format.
+`T.pp()` pretty prints inside a method chain when called on an object.
```

Comment:
I like the placing of `when called on an object` but  you'll need to retain the `inline` part, as that's an important part of the documentation. The method is called inline, rather than accepting a parameter.